Formula/portable-openssl.rb

```diff
@@ -39,6 +39,7 @@ def arch_args
       {
         :x86_64 => %w[linux-x86_64],
         :i386  => %w[linux-generic32],
+        :arm => %w[linux-armv4],
```

Comment:
It must be no newer than ARMv6, as the Raspberry Pi 1 has an ARMv6 processor.
See https://en.wikipedia.org/wiki/Raspberry_Pi#Processor